src/vs/workbench/electron-browser/main.contribution.ts

```diff
@@ -418,6 +418,28 @@ configurationRegistry.registerConfiguration({
 			'type': 'boolean',
 			'default': false,
 			'description': nls.localize('zenMode.restore', "Controls if a window should restore to zen mode if it was exited in zen mode.")
+		},
+		'zenMode.centerEditor': {
+			'type': 'boolean',
+			'default': false,
+			'description': nls.localize('zenMode.centerEditor', "Controls if a single editor is centered when in zen mode.")
+		},
+		'zenMode.centerEditorSize': {
```

Comment:
I'm in favor of a simpler approach where the max centered editor size is based on columns,  as with `editor.wordWrapColumn`